### PR TITLE
Fix test template to match created directories

### PR DESCRIPTION
**Pull request by soltysh**
@bparees found this while working on my app for PyCon, ptal

### CODE REVIEW

**Comment by soltysh** [comment]
I found one another in docker template as well.

File: pkg/create/templates/test.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ fi
 
 test_dir="$($READLINK_EXEC -zf $(dirname "${BASH_SOURCE[0]}"))"
 image_dir=$($READLINK_EXEC -zf ${test_dir}/..)
-scripts_url="file://${image_dir}/.s2i/bin"
+scripts_url="file://${image_dir}/s2i/bin"
```
Comment (bparees) [inline pkg/create/templates/test.go]: looks like this arg isn't even used in this file?

**Comment by soltysh** [comment]
Updated.

**Comment by bparees** [comment]
[merge]

**Pull request merged**